docs/contributing-to-airbyte/developing-on-docker.md

```diff
@@ -0,0 +1,46 @@
+# Developing on docker
+
+## Version
+
+This doc is relevant starting from the version 0.30.28-alpha. Prior to that, it was another design.
+
+## Incrementality 
+
+The docker build is fully incremental, which means that it will only build an image if it is needed. We need to keep it that way.
```

Comment:
It's worth clarifying that this is for the platform build.

Reply:
Done,